src/tools/cephfs/top/cephfs-top

```diff
@@ -1163,7 +1172,10 @@ class FSTop(FSTopBase):
             elif cmd == curses.KEY_END:
                 hscrollOffset = self.PAD_WIDTH - self.viewportWidth - 1
             elif cmd == curses.KEY_RESIZE:
-                # terminal resize event. Update the viewport dimensions
+                # terminal resize event.
+                # Update the pad dimensions
+                self.PAD_WIDTH = DEFAULT_PAD_WIDTH
```

Comment:
just a quick question, so when the terminal window is resized, the default pad width is 300 so does it also work with quite small screens?